### PR TITLE
[Old Website] update banner for steering group election 2024

### PR DESCRIPTION
**Pull request by bendhouseart**
Updates banner.

### CODE REVIEW

File: _includes/steering_group_election_section.html [hunk 1]
```diff
@@ -6,6 +6,13 @@
 
     <h2>BIDS steering group election</h2>
 
+    <p>
+      This year's election has started, please check your BIDS email for a link to the ballot on <a href="https://opavate.com" target="_blank">opavote.com</a> and cast your vote.
+      
+      your email for a notification from them. The polls will open on <b>Monday, October 21st</b> and close on <b>Friday, November 1st</b>.
```
Comment (bendhouseart) [inline _includes/steering_group_election_section.html:12]:
```suggestion
     The polls will open on <b>Monday, October 21st</b> and close on <b>Friday, November 1st</b>.
```

File: _includes/steering_group_election_section.html [hunk 2]
```diff
@@ -6,6 +6,13 @@
 
     <h2>BIDS steering group election</h2>
 
+    <p>
+      This year's election has started, please check your BIDS email for a link to the ballot on <a href="https://opavate.com" target="_blank">opavote.com</a> and cast your vote.
```
Comment (Remi-Gau) [inline _includes/steering_group_election_section.html:10]: - Add one sentence to tell people to contact us if they believe they should have got an email and did not or update the sentence below that explains how to contact us.
```suggestion
      This year's election has started. 
      If you are <a href="https://bids-specification.readthedocs.io/en/latest/appendices/contributors.html">a BIDS contributor</a>,
      please check your BIDS email for a link to the ballot on <a href="https://opavate.com" target="_blank">opavote.com</a> and cast your vote.
```

**Review by Remi-Gau — APPROVED** [review]
merge at will

**Comment by Remi-Gau** [comment]
> do you want to open a similar PR on the main branch for the new website just in case someone lands there?
> or so that we have it ready for next year

just saw the other PR

File: _includes/steering_group_election_section.html [hunk 3]
```diff
@@ -6,6 +6,15 @@
...
     <h2>BIDS steering group election</h2>
 
+    <h2><b>BIDS steering group election</b></h2>
+    This year's election has started. 
+    
+    If you are <a href="https://bids-specification.readthedocs.io/en/latest/appendices/contributors.html">a BIDS contributor</a>,
+    
+    please check your BIDS email for an <a href="https://opavote.com" target="_blank">opavote voting link</a> to cast your vote.
+    
+    The polls will open on <b>Monday, October 21st</b> and close on <b>Friday, November 1st</b>
```
Comment (bendhouseart) [inline _includes/steering_group_election_section.html:16]:
```suggestion
    <p>
    This year's election has started. 
    
    If you are <a href="https://bids-specification.readthedocs.io/en/latest/appendices/contributors.html">a BIDS contributor</a>,
    
    please check your BIDS email for an <a href="https://opavote.com" target="_blank">opavote voting link</a> to cast your vote.
    
    The polls will open on <b>Monday, October 21st</b> and close on <b>Friday, November 1st</b>
    </p>
```

**Pull request merged**